tests/test_pex.py

```diff
@@ -257,6 +259,105 @@ def test_pex_run():
       assert fake_stdout.read() == b'hellohello'
 
 
+class PythonpathIsolationTest(namedtuple('PythonpathIsolationTest',
+                                         ['pythonpath', 'dists', 'exe'])):
+
+  @staticmethod
+  def pex_info(inherit_path):
+    pex_info = PexInfo.default()
+    pex_info.inherit_path = inherit_path
+    return pex_info
+
+  def assert_isolation(self, inherit_path, expected_output):
+    env = dict(PYTHONPATH=self.pythonpath)
```

Comment:
Nit: collection literal preferred, per [Pants style guide](https://www.pantsbuild.org/styleguide.html#collection-literals). Not worth losing sleep over or fixing retroactively, though.

Reply:
Hrm, that definitely is not the style guide used in PEX today.String interpolation doesn't even apply for example due to PEX supporting 2.7. Even if it did though, this one I might have wanted to debate a little. For dicts in particular, dict can be viewed as nicer than a literal since the kwarg form used here puts sane restrictions on env var keys as well as making them appropriately "pop out" visually in syntax highlighters. Thats why I tend to use it for env dicts anyhow.